lib/builders/List.js

```diff
@@ -301,6 +301,7 @@ function applySelection (parent, definition) {
   } else if (selectionType === 'single') {
     option.addAttribute(':val', rowKey ? `item.${rowKey}` : 'item')
     option.addAttribute(':value', `${destPath}`)
+    option.addAttribute(':aria-label', `item.${'description'}`)
```

Comment:
This won't work using 'description' as not all data will have a 'description' property - this is the purpose of rowKey, if we are wanting to specify a property to use, otherwise it uses the whole object.

I think it would be better to do along the lines of `option.addAttribute(': aria-label', rowKey ? `item.${rowKey}` : 'item')`

Reply:
I have already tried this however the issue is it spits out the uprn and not the address when in a address search results

Reply:
Seeing as this is the List component, might it be best to use the label which is displayed for the aria label?
The label is specified in the component definition, so you can find it in definition.templates.label

https://github.com/wmfs/cardscript-to-quasar/blob/670a3a5850f3027a3e1105c400f612203937acdd/lib/builders/List.js#L60